covasim/population.py

```diff
@@ -28,6 +28,14 @@ class People(list):
     people with a certain attribute).
     '''
 
+    def __init__(self, people):
+        self.array = np.array(people)
```

Comment:
Is there a performance benefit of using an array rather than a list? I know it does make indexing easier (e.g. `sim.people[rel_susceptible == 0]` is cool, but not sure it's worth it for that one line!)

Reply:
It's more about being able to get slices of data. It's more than that one line, later I'm pulling list of susceptibles based on contact indices.